manage.sh

```diff
@@ -432,14 +434,72 @@ events_file_env(){
 	return 0
 }
 
+# Function that updates bitcoin.conf and Config.toml
+update_configs(){
+    if [ "${NETWORK}" == "testnet" ]; then
+		BTC_HOST=${TBTC_HOST}
+		BTC_RPC_USER=${TBTC_RPC_USER}
+		BTC_RPC_PASS=${TBTC_RPC_PASS}
+		BTC_RPC_PORT=${TBTC_RPC_PORT}
+		BTC_P2P_PORT=${TBTC_P2P_PORT}
+    fi
+	CONFIG_TOML="${SCRIPTPATH}/conf/${NETWORK}/Config.toml"
+	BTC_CONF="${SCRIPTPATH}/conf/${NETWORK}/bitcoin.conf"
+	${VERBOSE} && log "update_configs BTC_HOST: ${BTC_HOST}"
+	${VERBOSE} && log "update_configs BTC_RPC_USER: ${BTC_RPC_USER}"
+	${VERBOSE} && log "update_configs BTC_RPC_PASS: ${BTC_RPC_PASS}"
+	${VERBOSE} && log "update_configs BTC_RPC_PORT: ${BTC_RPC_PORT}"
+	${VERBOSE} && log "update_configs BTC_P2P_PORT: ${BTC_P2P_PORT}"
+	${VERBOSE} && log "update_configs CONFIG_TOML: ${CONFIG_TOML}"
+	${VERBOSE} && log "update_configs BTC_CONF: ${BTC_CONF}"
+	${VERBOSE} && log "update_configs NETWORK: ${NETWORK}"
+
+    ## update Config.toml with btc vars
+	[[ ! -f "${CONFIG_TOML}" ]] && cp "${CONFIG_TOML}.sample" "${CONFIG_TOML}"
+	${VERBOSE} && log "${COLYELLOW}Updating values in ${CONFIG_TOML} from .env${COLRESET}"
+    $(sed -i.tmp "
+		/^peer_host/s/.*/peer_host = \"${BTC_HOST}\"/; 
+		/^username/s/.*/username = \"${BTC_RPC_USER}\"/;
+		/^password/s/.*/password = \"${BTC_RPC_PASS}\"/;
+		/^rpc_port/s/.*/rpc_port = ${BTC_RPC_PORT}/;
+		/^peer_port/s/.*/peer_port = ${BTC_P2P_PORT}/;
+	" "${CONFIG_TOML}" 2>&1) || {
+        log_exit "Unable to update values in Config.toml file: ${COLCYAN}${CONFIG_TOML}${COLRESET}"
+    }
+    ${VERBOSE} && log "${COLYELLOW}Deleting temp Config.toml file: ${CONFIG_TOML}.tmp${COLRESET}"
+    $(rm "${CONFIG_TOML}.tmp" 2>&1) || {
+        log_exit "Unable to delete tmp Config.toml file: ${COLCYAN}${CONFIG_TOML}.tmp${COLRESET}"
+    }
+    ## update bitcoin.conf with btc vars
+	if check_flags "${FLAGS_ARRAY[*]}" "bitcoin"; then
+		[[ ! -f "${BTC_CONF}" ]] && cp "${BTC_CONF}.sample" "${BTC_CONF}"
+		${VERBOSE} && log "Matched bitcoin flag"
+		${VERBOSE} && log "${COLYELLOW}Updating values in ${BTC_CONF} from .env${COLRESET}"
+		$(sed -i.tmp "
+			/^rpcport/s/.*/rpcport=${BTC_RPC_PORT}/; 
+			/^rpcuser/s/.*/rpcuser=${BTC_RPC_USER}/;
+			/^rpcpassword/s/.*/rpcpassword=${BTC_RPC_PASS}/;
+			/^bind/s/.*/bind=0.0.0.0:${BTC_P2P_PORT}/;
+			/^rpcbind/s/.*/rpcbind=0.0.0.0:${BTC_RPC_PORT}/;
+		" "${BTC_CONF}" 2>&1) || {
+			log_exit "Unable to update values in bitcoin.conf file: ${COLCYAN}${BTC_CONF}${COLRESET}"
+		}
+		${VERBOSE} && log "${COLYELLOW}Deleting temp bitcoin.conf file: ${BTC_CONF}.tmp${COLRESET}"
+		$(rm "${BTC_CONF}.tmp" 2>&1) || {
+			log_exit "Unable to delete tmp bitcoin.conf file: ${COLCYAN}${BTC_CONF}.tmp${COLRESET}"
+		}
+	fi
+	return 0
+}
+
 # If bitcoin flag is detected when I'm starting the node then
 # the stacks blockchain needs to use a different Config.toml file
 # so it uses the local bitcoin node instead of the remote one
 setup_bitcoin() {
```

Comment:
outside of printing a verbose log item, what purpose does this function have?
unless it's being used for something, it should be removed

Reply:
None, it was scheduled to be removed. 
Removed in next commit.